File: uitesting/e2etestsmain5/pipelines/Workflow-Automated-GITHUB-1663914187899/code/workflowautomatedgithub1663914187899/graph/Filter_1.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from workflowautomatedgithub1663914187899.config.ConfigStore import *
from workflowautomatedgithub1663914187899.udfs.UDFs import *

def Filter_1(spark: SparkSession, in0: DataFrame) -> DataFrame:
    return in0.filter(((col("customer_id") > lit(10)) & col("first_name").like("%G%")))

File: uitesting/e2etestsmain5_external/pipelines/Workflow-Automated-GITHUB-1665636073408/code/src/main/scala/io/prophecy/pipelines/workflowautomatedgithub1665636073408/graph/Filter_1.scala
package io.prophecy.pipelines.workflowautomatedgithub1665636073408.graph

import io.prophecy.libs._
import io.prophecy.pipelines.workflowautomatedgithub1665636073408.config.ConfigStore._
import io.prophecy.pipelines.workflowautomatedgithub1665636073408.udfs.UDFs._
import io.prophecy.pipelines.workflowautomatedgithub1665636073408.udfs._
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object Filter_1 {

  def apply(spark: SparkSession, in: DataFrame): DataFrame =
    in.filter((col("customer_id") > lit(10)).and(col("first_name").like("%G%")))

}
